pkg/migrations/updater_test.go

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package migrations_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/xataio/pgroll/pkg/migrations"
+)
+
+func TestFileUpdater(t *testing.T) {
+	updater := migrations.NewFileUpdater()
+
+	t.Run("update with no operations", func(t *testing.T) {
+		rawMigration := &migrations.RawMigration{Operations: []byte("[]")}
+		migration, err := updater.Update(rawMigration)
+		if err != nil {
+			t.Fatalf("expected no error, got %v", err)
+		}
```

Comment:
Use `require.NoError(t, err)` rather than these `if` checks throughout this file.